test/unit/specs/users/cd-create-account.spec.js

```diff
@@ -0,0 +1,240 @@
+import vueUnitHelper from 'vue-unit-helper';
```

Comment:
A general comment on tests, I personally find it really useful for tests to have
```
// ARRANGE
// ACT
// ASSERT
```
to break them up and make them easier to read.